### PR TITLE
Bugfix/notifications api url

### PR DESCRIPTION
**Pull request by kovacshuni**
* tested in XP

### CODE REVIEW

File: notifications-push@.service [hunk 1]
```diff
@@ -17,13 +17,12 @@ ExecStart=/bin/sh -c '\
...
 	export NOTIFICATIONS_PUSH_PORT=8548; \
 	export GROUP_ID=$(/usr/bin/etcdctl get /ft/config/environment_tag)"-notifications-push-%i"; \
-	export API_BASE_URL="https://"$(/usr/bin/etcdctl get /ft/config/environment_tag)"-up.ft.com"; \
 	/usr/bin/docker run --rm --name %p-%i -p $NOTIFICATIONS_PUSH_PORT:8080 \
 		--env "QUEUE_PROXY_ADDRS=http://%H:8080" \
 		--env "GROUP_ID=$GROUP_ID" \
 		--env "TOPIC=CmsPublicationEvents" \
 		--env "QUEUE_HOST=kafka" \
-		--env "API_BASE_URL=$API_BASE_URL" \
+		--env "API_BASE_URL=http://api.ft.com" \
```
Comment (tosan88) [inline notifications-push@.service]: Can you please construct the url based on the env variable? Example: https://github.com/Financial-Times/up-service-files/blob/master/document-store-api%40.service#L20

Comment (kovacshuni) [inline notifications-push@.service]: Done.

**Comment by Nesze** [comment]
👍

**Comment by tosan88** [comment]
:+1:

**Comment by khatton-ft** [comment]
:+1:

**Comment by catalin-ilea** [comment]
lgtm

**Pull request merged**